vmdb/db/migrate/20150401223517_container_nodes_hardware.rb

```diff
@@ -0,0 +1,16 @@
+class ContainerNodesHardware < ActiveRecord::Migration
+  def change
+    add_column :hardwares, :container_node_id, :bigint
+
+    # identity_infra is the node id assigned by the infrastructure
+    add_column :container_nodes, :identity_infra, :string
+
+    # identity_machine is the machine id generated by systemd:
+    # http://www.freedesktop.org/software/systemd/man/machine-id.html
+    add_column :container_nodes, :identity_machine, :string
+
+    # identity_system is the system uuid collected by the bios, in
+    # some infrastructure is injected by the hypervisor
+    add_column :container_nodes, :identity_system, :string
```

Comment:
Except if we switch from the `ContainerNode` to `Hardware` relationship in favor of `ContainerNode` to `ComputerSystem`, in which case the `container_node_id` in `:hardwares` would be removed.